server/modules.json

```diff
@@ -61,5 +52,14 @@
         "notifications": 0,
         "purchased": false,
         "purchasable": true
+    },
+    "modules": {
+        "id": 5,
+        "title": "Add Modules",
+        "icon": "mdi-cog",
+        "to": "/modules",
+        "notifications": 0,
+        "purchased": true,
+        "purchasable": false
```

Comment:
moved Add Modules to the bottom. Please keep it here.

Reply:
Let's add a comment in-code, something like 
//all future modules should go above this one

Reply:
I'm not sure if we can add that in a JSON file?

Reply:
ahhhhhhh good call.